### ISSUE TITLE
Bad attachments breaks all steps of batch

### ISSUE THREAD

**Issue by admizh** [reportportal/reportportal#19]
RP have default value for batch: rp.batch.size.logs == 10, When we have wrong logging event (try send big file) - all steps from this batch is lost in RP.
perhaps it will be fixed by: #16 

```java
    @Test
    public void bigFileTest() throws Exception {
        //rp.batch.size.logs = 10 (default value for batch)
        URL resource = Resources.getResource("big.file");
        String path = Paths.get(resource.toURI()).toAbsolutePath().toString();
        logger.info("1# event before sending big file");
        logger.info("2# event before sending big file");
        logger.info("3# event before sending big file");
        logger.info("4# event before sending big file");
        logger.info("5# event before sending big file");
        logger.info("RP_MESSAGE#FILE#{}#{}", path, "7# event (very big file)");
        logger.info("8# event after sending big file");
        logger.info("9# event after sending big file");
        logger.info("10# event after sending big file"); //will triggered batch call to RP
    }
```

![2016-12-02_14-04-23](https://cloud.githubusercontent.com/assets/6014483/20830064/4bbc1efa-b898-11e6-9ff6-20a62e1cc9b8.png)

**Comment by avarabyeu** [comment]
@admizh Good catch!

**Comment by avarabyeu** [comment]
@admizh the problem here is that we are limiting whole request size. Not sure we need to allow users to upload a big requests.

**Comment by e-olga** [comment]
Added to RP backlog

**Comment by DzmitryHumianiuk** [comment]
@admizh @avarabyeu 
possible solutions: add as parameter for server (via docker compose)

planned sprint12

**Comment by avarabyeu** [comment]
Fixed in 3.0. Now there are following configuration properties for service-api are available:
rp.upload.maxUploadSize=128MB
rp.upload.maxFileSize=32MB

KB,MB,GB postfixes are allowed.
